hledger-lib/Hledger/Read/RulesReader.hs

```diff
@@ -730,9 +731,10 @@ maybeNegate :: MatcherPrefix -> Bool -> Bool
 maybeNegate Not origbool = not origbool
 maybeNegate _ origbool = origbool
 
--- | Given the conversion rules, a CSV record and a hledger field name, find
--- the value template ultimately assigned to this field, if any, by a field
--- assignment at top level or in a conditional block matching this record.
+-- | Given the conversion rules, a CSV record and a hledger field name, collect
+-- the value templates (and their parent Conditional Blocks where applicable)
+-- assigned to this field, if any, by top-level field assignments and
+-- conditional blocks matching this record.
```

Comment:
Excuse quibbling, it's because I expect I'll be staring confusedly at this file in future:

"find the value template" changed to "collect the value templates (and their parent Conditional Blocks where applicable)".

but the type signature looks more like "find a single template or its parent conditional block" ?

Reply:
You're absolutely right; I forgot that `getEffectiveAssignment` applied `lastMay` to the list, I thought it was done by its callers (it was in an earlier revision of mine). I'll address it.